### DISCUSSION TITLE
Playlist Shuffle Mode:  How to stop shuffling different song titles and artists from same album?

### DISCUSSION THREAD

**Discussion by kkabbers** [AzuraCast/AzuraCast#6000] [Q&A]
Is there a mechanism to control playlist shuffling different artists and song titles from same album? My playlist are all on shuffle mode and it is doing a beautiful job but it does frequently select different songs from the same album. Is there a way to prevent this?

**Comment by Moonbase59** [comment]
To fulfill different regulations in different countries, we might need more granular playout restriction options, see https://features.azuracast.com/suggestions/364423/more-granular-playout-restrictions-so-local-regulations-can-be-kept.
